app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -163,12 +202,44 @@ def parse_deployment_name(name)
         name.sub(/^.*deployment=/, '')
       end
 
-      def parse_middleware_server(eap)
+      def parse_server_group_name(name)
+        name.sub(/^Domain Server Group \[/, '').chomp(']')
+      end
+
+      def parse_domain_server_name(name)
+        name.sub(/^.*\/server=/, '')
+      end
+
+      def parse_middleware_domain(domain)
```

Comment:
done in commit d95018b0a, I've also added the `feed` and `properties` columns to entities that were missing it.